db/migrations/20240514113500_create_cnb_lifecycle_data.rb

```diff
@@ -38,10 +37,10 @@
 
     transaction do
       alter_table(:buildpack_lifecycle_buildpacks) do
-        drop_foreign_key [:cnb_lifecycle_data_guid]
+        drop_foreign_key [:cnb_lifecycle_data_guid] if @db.foreign_key_list(:buildpack_lifecycle_buildpacks).any? { |fk| fk[:name] == :fk_blcnb_bldata_guid }
```

Comment:
@johha But isn't that the same as for `add_foreign_key` (line 22)? Just in a different order? Should/could we do it in the same way (if -> alter_table).

Reply:
rubocop wanted it that way...

Reply:
Good argument :)

Reply:
I'll change it, rubocop actually complained about something else